src/main/java/com/cinema/dao/impl/TicketDaoImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.cinema.dao.impl;
+
+import com.cinema.dao.TicketDao;
+import com.cinema.exception.DataProcessingException;
+import com.cinema.lib.Dao;
+import com.cinema.model.Ticket;
+import com.cinema.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class TicketDaoImpl implements TicketDao {
+    @Override
+    public Ticket add(Ticket ticket) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.persist(ticket);
```

Comment:
let's be consistent and use the same method everywhere unless there is a particular reason to use a specific method
I think that `save()` is suitable for every DaoImpl class